glossary.md

```diff
@@ -18,3 +18,7 @@ You may also need to check <https://chromium.googlesource.com/chromiumos/docs/+/
   primarily administrated by Matt Godbolt.
 * primordials: Pristine built-ins that are not affected by prototype pollution
   and tampering with built-ins.
+* undici: An HTTP/1.1 client, written from scratch for Node.js. Undici means
+  eleven in Italian. 1.1 -> 11 -> Eleven -> Undici. It is also a Stranger Things
+  reference. Undici is maintained within a separate repository, see more details
+  on the [undici repository](https://github.com/nodejs/undici).
```

Comment:
Duplicating the word's origin here does not seem useful. The link already explains the origin, and this seems to be a verbatim copy, which means that it will be inconsistent eventually.

Reply:
I would replace "the" in "the HTTP client used in Node.js" with "an alternative" because there is also the `http` module.

Reply:
is "An HTTP 1.1 client used in Node.js" ok? undici is an HTTP 1.1. client. but if undici will include HTTP 2.0 part in the future, the leave 1.1 out will be better.